buildSrc/src/main/kotlin/toolkit-intellij-subplugin.gradle.kts

```diff
@@ -169,6 +169,7 @@ tasks.runIde {
     systemProperty("aws.toolkit.developerMode", true)
     systemProperty("ide.plugins.snapshot.on.unload.fail", true)
     systemProperty("memory.snapshots.path", project.rootDir)
+    systemProperty("idea.auto.reload.plugins", false)
```

Comment:
What is this required for?

Reply:
stops annoying plugin reloading all the time